plugins/ingest-attachment/src/test/java/org/elasticsearch/ingest/attachment/AttachmentProcessorTests.java

```diff
@@ -214,6 +214,10 @@ public void testAsciidocDocument() throws Exception {
         assertThat(attachmentData.get("content_type").toString(), containsString("text/plain"));
     }
 
+    public void testZipFileDoesNotHang() throws Exception {
```

Comment:
can you add a comment here linking to the apache-issue?

Reply:
Added.